dpnp/dpnp_iface_arraycreation.py

```diff
@@ -171,14 +175,64 @@ def array(obj, dtype=None, copy=True, order='C', subok=False, ndmin=0):
         checker_throw_value_error("array", "copy", copy, True)
 
     if order != 'C':
-        checker_throw_value_error("array", "order", order, 'K')
+        checker_throw_value_error("array", "order", order, 'C')
 
     if ndmin != 0:
         checker_throw_value_error("array", "ndmin", ndmin, 0)
 
     return dpnp_array(obj, dtype)
 
 
+def asanyarray(a, dtype=None, order=None):
```

Comment:
Could you please add block `Limitations` to all the docstring something like this:
```
    Limitations
    -----------
    Parameter ``order`` is supported only with default value `'C'`.
```